operator/src/main/java/oracle/kubernetes/operator/steps/HttpRequestProcessing.java

```diff
@@ -35,8 +35,13 @@
   }
 
   static HttpAsyncRequestStep createRequestStep(HttpRequest request, HttpResponseStep responseStep) {
+    return createRequestStep(request, responseStep, HTTP_TIMEOUT_SECONDS);
+  }
+
+  static HttpAsyncRequestStep createRequestStep(HttpRequest request, HttpResponseStep responseStep,
```

Comment:
Since the `withTimeoutSeconds` method is public, have the caller use that rather than creating an additional method that does the same thing.

Reply:
removed the additional method and had caller directly use the call that includes `withTimeoutSeconds` method